material-overrides/assets/stylesheets/tanssi.css

```
@@ -679,24 +679,66 @@ details.function,
 }
 
 /* Feedback widget styling */
-.md-feedback {
-  margin: 2em auto;
-  text-align: center;
+.feedback-actions-container {
+  margin-top: 4rem;
```

```suggestion
  margin: 3rem .5rem 0;
```

This margin makes it look a little bit better IMO